src/modules/endpoint/endpoints/__tests__/categories.spec.js

```diff
@@ -0,0 +1,84 @@
+import categories from '../categories'
+import CategoriesContainer from '../../models/CategoriesContainer'
+import CategoryModel from '../../models/CategoryModel'
+
+describe('categories', () => {
+  const categoriesJSON = [{
+    id: 3650,
+    permalink: {
+      url_page: 'anlaufstellen'
+    },
+    title: 'Anlaufstellen zu sonstigen Themen',
+    status: 'publish',
```

Comment:
Add a page with status 'trash' to test the filter.

Reply:
okay